tests/karma/unit/services/lineage-model-generator.js

```diff
@@ -93,6 +128,34 @@ describe('LineageModelGenerator service', () => {
       });
     });
 
+    it('hydrates lineage contacts - #3812', () => {
```

Comment:
is that the same test as l.64? Name is the same, looks roughly the same. Copy-paste accident?